tests/x11/sshxterm.pm

```diff
@@ -23,6 +23,7 @@ sub run {
     my ($self) = @_;
     mouse_hide(1);
     x11_start_program('xterm');
+    enter_cmd("ssh-keygen -f ~/.ssh/id_rsa -y > ~/.ssh/id_rsa.pub");
```

Comment:
I think you'd better to add comments for the reason of re-create the public key here.

Reply:
Maybe an product issue after migration?